WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -9,9 +10,13 @@ def bad_api_parameter(e)
 
   DEFAULT_API_RESULT_LIMIT = 20
 
+  def current_resource_owner
```

Comment:
I prefer `current_api_user`, since many don't know what Doorstop is.

The one problem with `||=` caching is that it doesn't work when a possible value is `nil` or `false`.